.circleci/config.yml

```diff
@@ -15,17 +15,15 @@ jobs:
           command: git submodule update --init --recursive
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "Gemfile.lock" }}
-            # fallback to using the latest cache if no exact match is found
-            - v1-dependencies-
+            - v1-dependencies-generator
       - run:
           name: install dependencies
           command: |
             bundle install --jobs=4 --retry=3 --path vendor/bundle
```

Comment:
```suggestion
            bundle install --jobs=4 --retry=3 --path .vendor/bundle && bundle clean
```

Reply:
Added `bundle clean` because of the note [here](https://circleci.com/docs/2.0/caching/#bundler-ruby).

Reply:
Also fixes the path so bundler installs dependencies in the same directory that is being cached.

Reply:
Interesting. I wonder why they don't suggest that during the setup. I just copy pasted this config. I suspect that api common protos repo I setup awhile back has this same cache path issue.

Reply:
This still doesn't install `minitest-focus` when I run it locally. If I `cd` into `gapic generator` and do an install it's fine, but it doesn't work when doing an install at the root only.